tsconfig.json

```diff
@@ -12,11 +12,13 @@
     "strict": true,
     "noImplicitAny": false,
     "outDir": "./dist",
+    "rootDir": "./src",
```

Comment:
Maybe make it just the root to keep structure?
```suggestion
    "rootDir": ".",
```

Reply:
The the output will be `dist/src/...` instead of `dist/...`